src/js/datetimepicker.js

```diff
@@ -113,7 +165,12 @@
         '           <td colspan="7" >' +
         '              <span    class="{{ data.currentView }}" ' +
         '                       data-ng-repeat="dateObject in data.dates"  ' +
-        '                       data-ng-class="{active: dateObject.active, past: dateObject.past, future: dateObject.future, disabled: !dateObject.selectable}" ' +
+        '                       data-ng-class="{ ' +
+        '                          active: dateObject.active, ' +
+        '                          past: dateObject.past, ' +
+        '                          future: dateObject.future, ' +
+        '                          today: dateObject.isCurrent(data.currentView, showCurrent, configuration.minuteStep), ' +
```

Comment:
to be consistent with the rest of the API, this should be 
today: dateObject.today 

the today property should be set when the date objects are calculated - no need to delay this calculation until the view expressions are evaluated.